c/ldv-challenges/linux-3.14__complex_emg__linux-alloc-spinlock__drivers-media-rc-imon_true-unreach-call.cil.c

```diff
@@ -4218,7 +4218,10 @@ __inline static unsigned int __create_pipe(struct usb_device *dev , unsigned int
 extern void kfree(void const   * ) ;
 __inline static void *kmalloc(size_t size , gfp_t flags ) ;
 __inline static void *kzalloc(size_t size , gfp_t flags ) ;
-extern struct input_dev *input_allocate_device(void) ;
+struct input_dev *input_allocate_device(void) {
+       return kzalloc(sizeof(struct input_dev), 0x10u | 0x40u | 0x80u);
```

Comment:
what is the meaning of 0x10u | 0x40u | 0x80u ?

Reply:
for GFP_KERNEL I suggest to use 208U as in the other parts of the code

Reply:
Yes, the meaning of `0x10u | 0x40u | 0x80u` was exactly `GFP_KERNEL`. Didn't know that `208U` is used, will fix it.